community-plugins.json

```
@@ -4185,5 +4185,12 @@
         "description": "Convert Markdown files to BBCode",
         "repo": "salockhart/obsidian-bbcode",
         "branch": "master"
+    },
+    {
+        "id": "obsidian-meta-title-filename-plugin",
+        "name": "Meta title filenames",
+        "author": "Snezhig",
+        "description": "Let you use data from meta block as a filename",
```

```suggestion
        "description": "Lets you define a title in frontmatter to be displayed as the filename",
```

Maybe something like this is more descriptive?